frontends/ol-components/src/components/TiptapEditor/ArticleEditor.tsx

```diff
@@ -190,13 +190,10 @@ const ArticleEditor = ({ onSave, readOnly, article }: ArticleEditorProps) => {
       file,
       async (file: File, progressCb?: (percent: number) => void) => {
         try {
-          const response = await uploadImage.mutateAsync({
-            file,
-            onUploadProgress: (e) => {
-              const percent = Math.round((e.loaded * 100) / (e.total ?? 1))
-              progressCb?.(percent)
-            },
-          })
+          uploadImage.setNextProgressCallback(progressCb)
```

Comment:
So the reason the error happens is:
1. Tanstack query devtools are a chrome extension, so they run in a webworker
2. The devtools track mutation state, which includes the args we pass to `mutate` (or `mutateAsync`).
3. In order for that data to get into the webworker, the data must be serializable. Previously we were passing a function, which was not serializable. 

Now `onUploadProgress` is no longer an arg of mutateasync, which avoids the issue.

I don't love changing the application code to avoid a devtool error. However, it is useful to not have these errors showing up, and this alternative implementation seems fine.